sdk/storage/Swagger/Queues/readme.md

```diff
@@ -0,0 +1,23 @@
+# Queue Storage
+> see https://aka.ms/autorest
+
+## Configuration
+``` yaml
+
+# Prevent swagger validation because it complains about vendor extensions
+# instead of ignoring them per the spec
+pipeline:
+  swagger-document/individual/schema-validator:
+     scope: unused
+
+# Generate queue storage
+input-file: ./queue.json
+output-folder: ../../Azure.Storage.Queues/src

```

Comment:
I'd prefer to have the generated files in a separate folder from non-generated files.  It allows them to be wiped easily as part of normal regeneration, which keeps things clean when removing/renaming things.  Otherwise, they will accumulate in the repository.

Reply:
I'll add a Generated folder under /src